lib/translations_v2.py

```diff
@@ -30,7 +30,7 @@
     "he": (":ריוואה גזמ תיזחת",             "", "", "", "", "", "", "", ""),
     "hi": ("मौसम की जानकारी",               "मौसम", "समय मण्डल", "अभी", "उदय", "सूर्योदय", "चरम बिन्दु", "सूर्यास्त", "संध्याकाल"),
     "hr": ("Vremenska prognoza za:",        "", "", "", "", "", "", "", ""),
-    "hu": ("Időjárás előrejelzés:",         "", "", "", "", "", "", "", ""),
+    "hu": ("Időjárás előrejelzés:",         "Időjárás", "időzóna", "aktuális", "hajnal", "napkelte", "dél", "naplemente", "szürkület"),
```

Comment:
```suggestion
    "hu": ("Időjárás előrejelzés:",         "Időjárás", "Időzóna", "Aktuális idő", "Hajnal", "Napkelte", "Dél", "Naplemente", "Szürkület"),
```

Reply:
- I suggest to use capital casing, as it is in other lanuages.
- _aktuális_ means "current" or "actual", whereas _aktuális idő_ means "current time"